lightning/src/util/config.rs

```diff
@@ -151,6 +156,7 @@ impl Default for ChannelHandshakeLimits {
 	fn default() -> Self {
 		ChannelHandshakeLimits {
 			min_funding_satoshis: 0,
+			max_funding_satoshis: 1 << 24,
```

Comment:
Maybe you can import `MAX_FUNDING_SATOSHIS_NO_WUMBO`.

Reply:
Ended up getting rid of this constant